index.php

```diff
@@ -8,12 +8,14 @@
 require_once 'autoload.php';
 
 use Mautic\CoreBundle\ErrorHandler\ErrorHandler;
+use Mautic\CoreBundle\Loader\EnvironmentHandler;
 use Mautic\Middleware\MiddlewareBuilder;
 use Symfony\Component\HttpFoundation\Request;
 
-ErrorHandler::register('prod');
+$config = (new EnvironmentHandler())->getEnvParameters();
```

Comment:
```suggestion
require dirname(__DIR__).'/config/bootstrap.php';
```
Let's load the environmental variables the Symfony way. See https://github.com/symfony/recipes/blob/main/symfony/framework-bundle/4.4/public/index.php#L7 and https://github.com/symfony/recipes/blob/main/symfony/framework-bundle/4.4/config/bootstrap.php

Reply:
Hey @fedys, thank you so much for reviewing, I'm reading and trying to understand, and it's clear. 

I have a question, the item 1, when you say similar, are you talking about parts of code with environment right?

Reply:
Correct 👍

Reply:
@fedys is it a good idea to include the lines that [handle proxies](https://github.com/symfony/recipes/blob/41d2cf13ea897a08b0f376d20011a69a73ec0366/symfony/framework-bundle/4.4/public/index.php#L15) into this PR or make a new PR for them?

Reply:
I would stick with changes connected with the environmental variables in this PR. Let's create a separate PR for proxies related code.